src/components/page-details.jsx

```diff
@@ -86,15 +86,16 @@ export default class PageDetails extends React.Component {
         <div className="row">
           <div className="col-md-9">
             <h2 className="page-title">
-              <a
-                className="diff_page_url"
-                href={this.state.page.url}
-                target="_blank"
-                rel="noopener"
-              >
-                {this.state.page.title}
-              </a>
+              {this.state.page.title}
             </h2>
+            <a
+              className="diff_page_url"
+              href={this.state.page.url}
+              target="_blank"
+              rel="noopener"
+            >
+              {this.state.page.url}
+            </a>
```

Comment:
Now that this link needs to stay associated with the `<h2>`, it might be a good idea to wrap them together in a `<header>` (or replace the `<div>` that is wrapping them with a `<header>`).

Reply:
Unfortunately, the wrapping `<div>` still uses styles from bootstrap for layout. Removing that is a whole 'nother can of worms #87.

I do like the idea of moving towards more semantic markup, which is not a habit I've picked up yet. (When I was doing web dev as a job, html5 elements were still new and my ex-company didn't like the scary, different thing.) I added `<header>`, which would be more logical to style against in the future.